e2e/app.e2e.ts

```diff
@@ -151,5 +152,25 @@ describe('bterm launch', function() {
       .then(() => this.app.client.browserWindow.isMaximized())
       .then(result => expect(result).to.be.true);
   });
+
+  it('should open and close multi window of the app', () => {
+    let appToClose = clone(this.app);
```

Comment:
New cloned object creates when started creates new window. Yes, I did try this locally.